src/renderer/uia/UiaRenderer.cpp

```diff
@@ -246,7 +255,6 @@ UiaEngine::UiaEngine(IUiaEventDispatcher* dispatcher) :
     _selectionChanged = false;
     _textBufferChanged = false;
     _cursorChanged = false;
-    _prevSelection.clear();

```

Comment:
???

Reply:
wots all this then eh

Reply:
Well, I realized, why should we clear the previousSelection between frames? If it's clear when we render a new frame, we'll always have the selection be "new". So, this should reduce the number of times we send a selection event too.